schema_based_on_diagram.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE patients (
+id INT,
+name VARCHAR,
+PRIMARY key(id)
+);
```

Comment:
- [ ] Please add `date_of_birth`  column in the `patients` table as it is requested in the diagram. 

![image](https://user-images.githubusercontent.com/83803414/179783898-bed298d5-6ceb-4ca2-b9a2-76ba5940816b.png)